README.md

```diff
@@ -40,29 +40,51 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Set up MATLAB
-        uses: matlab-actions/setup-matlab@v1
+        uses: matlab-actions/setup-matlab@v2-beta
       - name: Run tests and generate artifacts
         uses: matlab-actions/run-tests@v1
         with:
           test-results-junit: test-results/results.xml
           code-coverage-cobertura: code-coverage/coverage.xml
 ```
 
+### Run MATLAB Script on GitHub-Hosted Runner
+Set up a GitHub-hosted runner to run the commands in a file named `myscript.m` in the root of your repository. To run the script, include the [Run MATLAB Command](https://github.com/matlab-actions/run-command/) action in your workflow.
+
+```yaml
+name: Run MATLAB Script on GitHub-Hosted Runner
+on: [push]
+jobs:
+  my-job:
+    name: Run MATLAB Script
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - name: Set up MATLAB
+        uses: matlab-actions/setup-matlab@v2-beta
+      - name: Run script
+        uses: matlab-actions/run-command@v1
+        with:
+          command: myscript
+```
+
 ## Set Up MATLAB
 When you define your workflow in the `.github/workflows` directory of your repository, specify the **Setup MATLAB** action as `matlab-actions/setup-matlab@v1`. The action accepts an optional input.
 
 | Input     | Description |
 |-----------|-------------|
-| `release` | (Optional) MATLAB release to set up. You can specify R2020a or a later release. If you do not specify `release`, the action sets up the latest release of MATLAB.<br/>**Example**: `R2021a`
-
+| `release` | (Optional) MATLAB release to set up. You can specify R2020a or a later release. If you do not specify `release`, the action sets up the latest release of MATLAB.<br/>**Example**: `R2022a`
+| `products` | (Optional) Space-separated list of products to install. If a product name contains white-space characters, replace them with underscores. For the full list of available products and their names, see [Products and Services](https://www.mathworks.com/products.html). By default, the task installs MATLAB and the Parallel Computing Toolbox.<br/> **Example**: `MATLAB Simulink`
```

Comment:
You say for a full list of products, see ...
Are transformational products also supported with mpm?

Reply:
Yeah, transformational products are supported with mpm. They can install them with this action and provide their own matlab-batch token  that we issued to them according to their license. Transformational products will not be licensed for Public repos though.

Reply:
Ok. So any changes to this paragraph in the beta documentation?

https://github.com/matlab-actions/setup-matlab#:~:text=The%20Setup%20MATLAB%20action%20is%20not%20supported%20on%20self%2Dhosted%20runners.%20Currently%2C%20it%20is%20available%20only%20for%20public%20projects.%20It%20does%20not%20set%20up%20transformation%20products%2C%20such%20as%20MATLAB%20Coder%E2%84%A2%20and%20MATLAB%20Compiler%E2%84%A2.

Reply:
The Setup MATLAB action is not supported on [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) runners. Currently, it is available only for public projects. It does not set up transformation products, such as MATLAB Coder™ and MATLAB Compiler™.

Reply:
Good catch. I updated some of the text in the intro.